content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: 'Environment variables reference'
+metaTitle: 'Prisma environment variables reference'
+metaDescription: 'This page gives an overview of all environment variables avalaible for use.'
+tocDepth: 2
+---
+
+## Overview
+
+This document describes different environment variables and their use cases.
+
+### DATABASE_URL
+
+`DATABASE_URL` is used to connect to your database. The url format can differ depending on the database it is connecting to.
+Prisma currently supports 4 types of database connectors, of which 3 use the `DATABASE_URL` environment variable. (SQLite sources a file path)
+
+- PostgreSQL
+- MySQL
+- SQLite
+- SQLServer
+
+Example using a PostgreSQL connector:
+
+```js file=.env
+DATABASE_URL = 'postgresql://USER:PASSWORD@HOST:PORT/DATABASE'
+```
+
+See [Connection URLs](../database-reference/connection-urls) for a breakdown of each database connector.
+
+### DEBUG
+
+`DEBUG` is used to enable debugging output in the `Prisma Client`.
```

Comment:
I would not use backticks when spelling out the "brand name" of Prisma Client. If we refer to the "technical" `PrismaClient` type/constructor then I think backticks are appropriate when spelled as a proper noun in two words I'd probably omit them.
```suggestion
`DEBUG` is used to enable debugging output in the `Prisma Client`.
```

```suggestion
`DEBUG` is used to enable debugging output in the Prisma Client.
```

Reply:
Good shout, it would be Prisma client then wouldnt it? As Prisma is the brand name that is capitalized

Reply:
Haha we're already getting into the weeds 😄 

> it would be Prisma client then wouldnt it

Not quite, "Prisma Client" is a brand name / proper noun itself. So are "Prisma Migrate" and "Prisma Studio". We also don't use articles on these, so it's never "The Prisma Client" but only "Prisma Client ". (e.g. "~The~ Prisma Client gets generated when you call `prisma generate`")

Reply:
Ok nice, thanks for the explanation!